sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -345,6 +345,7 @@ private void delay(long delay) {
         }
         try {
             if (delay > 0) {
+                logger.debug("Sleeping for: " + delay + " | Thread name: " + Thread.currentThread().getName());
```

Comment:
Use
```suggestion
                if (logger.isEnabledForLevel(LogLevel.DEBUG)) {
                    logger.debug("Sleeping for: " + delay + " | Thread name: " + Thread.currentThread().getName());
                }
```
to avoid unnecessary argument evaluation when `DEBUG` is not enabled.

Reply:
`isEnabledForLevel` gets called in `logger.debug` so I don't know if I need to do it here. I saw it being used in a couple more places so I'll change it anyway.